Demo/ViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class CollectionCell: UICollectionViewCell {
```

Comment:
I think it's probably better if we have Unsplash in a separate repo, too many dependencies. It's probably better if here we have a very lightweight demo.